src/main/resources/upgrade-files/4.1.1_5.0/rice_server/rice-server-script_mod7.xml

```diff
@@ -546,14 +566,17 @@
 		)
 		]]></sql>
 		<sql splitStatements="false"><![CDATA[
-		INSERT /*+ PARALLEL(KREW_RULE_TMPL_ATTR_T,4) */ INTO KREW_RULE_TMPL_ATTR_T
+		INSERT /*+ APPEND PARALLEL(4) */ INTO KREW_RULE_TMPL_ATTR_T
 		(RULE_TMPL_ATTR_ID, RULE_TMPL_ID, RULE_ATTR_ID, REQ_IND, ACTV_IND, DSPL_ORD, DFLT_VAL, VER_NBR, OBJ_ID)
-		SELECT /*+ PARALLEL(TEMP_KREW_RULE_TMPL_ATTR_T,4) */
+		SELECT /*+ PARALLEL(,4) */
```

Comment:
What is this extra comma doing here (just before the 4)?
`SELECT /*+ PARALLEL(,4) */`